pkg/clamav/clamav_test.go

```diff
@@ -3,18 +3,23 @@ package clamav
 import (
 	"testing"
 
+	"golang.org/x/net/context"
 	"github.com/openshift/clam-scanner/pkg/clamav"
 )
 
 type fakeClamSession struct {
 	t                  *testing.T
+	ctx                context.Context
 	path               string
+	filter             clamav.FilterFiles
```

Comment:
filter related items but I'm not objecting to this since the interface items were removed.  Just a note